index.html

```diff
@@ -4007,6 +4007,317 @@ <h2>
     </section>
   </section>
 
+  <section class="informative">
+    <h1>
+      Examples
+    </h1>
+    <section class="informative">
+      <h2>
+        DID Documents
+      </h2>
+
+      <p>
+        See <a href="https://www.w3.org/TR/did-spec-registries/#verification-method-types">did-spec-registries</a> for optional extensions and other verifcation method types.
+      </p>
+
+      <p class="note">
+        These examples are for information purposes only, 
+        it is considered a best practice to avoid using the same verification method for multiple purposes.
```

Comment:
Great to see this statement in here! This will prove incredibly valuable I expect.